From 847bf22860d949d98281444967971fd6c8d47a55 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:55 -0400
Subject: [redhat] kabi: Adding symbol cpu_present_mask (kernel/cpu.c)

Message-id: <8ced88c38ce493f449c1d13fd8f35c0ce231b2fe.1410510773.git.poros@redhat.com>
Patchwork-id: 95932
O-Subject: [PATCH RHEL7.1 055/233] kabi: Adding symbol cpu_present_mask (kernel/cpu.c)
Bugzilla: 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    This exported struct is not accessed directly, it gets accessed via a
    #define num_present_cpus which is used to find number of cpus on smp system.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/cpu_present_mask
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/cpu_present_mask
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/cpu_present_mask

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/cpu_present_mask b/redhat/kabi/kabi-module/kabi_ppc64/cpu_present_mask
new file mode 100644
index 0000000..ae8458d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/cpu_present_mask
@@ -0,0 +1,2 @@
+#1-
+0xfbdd4c01 cpu_present_mask vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/cpu_present_mask b/redhat/kabi/kabi-module/kabi_s390x/cpu_present_mask
new file mode 100644
index 0000000..e5888aa
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/cpu_present_mask
@@ -0,0 +1,2 @@
+#1-
+0x341cbed2 cpu_present_mask vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/cpu_present_mask b/redhat/kabi/kabi-module/kabi_x86_64/cpu_present_mask
new file mode 100644
index 0000000..7a6d851
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/cpu_present_mask
@@ -0,0 +1,2 @@
+#1-
+0x6b372259 cpu_present_mask vmlinux EXPORT_SYMBOL
-- 
1.7.1